src/Facility.Core/NewtonsoftJsonServiceSerializer.cs

```diff
@@ -111,6 +111,18 @@ private static void ToJsonWriter(object? value, JsonWriter jsonWriter)
 		return FromJsonTextReader(textReader, type);
 	}
 
+	/// <summary>
+	/// Deserializes a value from the serialization format.
+	/// </summary>
+	/// <remarks>Copies the stream content into a buffer to avoid blocking IO.</remarks>
+	public override async Task<object?> FromStreamAsync(Stream stream, Type type, CancellationToken cancellationToken)
```

Comment:
I think this would obsolete `ForceAsyncIO`?

Reply:
That was the idea. It probably should support a MemoryStreamCreator, but I wonder if it'll actually ever get used since the STJ serializer becoming the default.

Reply:
Yeah, if this is the only code path that needs a memory stream, might not be worth it.

Reply:
I guess `ServiceSerializerHttpContentSerializer` still uses memory stream; maybe we should just add a dependency to `Microsoft.IO.RecyclableMemoryStream` and use it directly (pit of success and all that). With a static method to swap it out in case people want to share with other libraries?

Or maybe `DelegateHttpContent` could hang on to the DTO and serialize directly to the stream? It seems like I had weird issues when I didn't override `TryComputeLength` though (without which `Content-Length` isn't set IIRC).